execution/inc/leatherman/execution/execution.hpp

```diff
@@ -430,6 +434,7 @@ namespace leatherman { namespace execution {
      */
     std::tuple<std::string, std::string> process_streams(
             bool trim,
+            bool strip_cr,
```

Comment:
Technically, this is an API change, but I can't imagine that anyone actually uses `process_streams` directly.

I'd actually consider moving this into an internal header "soon" to be a pretty good plan, since it's an implementation detail of the platform-specific execution codepaths